File: Android/src/main/java/com/tazkiyatech/quran/sdk/model/SectionMetadata.kt
package com.tazkiyatech.quran.sdk.model

/**
 * Metadata which describes a section of the Quran.
 *
 * @property sectionType The type of the section.
 * @property sectionNumber The number of this section within its section type.
 * @property numAyahs The number of Ayahs that this section consists of.
 * @property surahNumber The Surah number of the first verse in this section.
 * @property ayahNumber The Ayah number of the first verse in this section (relative to the Surah).
 */
data class SectionMetadata(
    val sectionType: String,
    val sectionNumber: Int,
    val numAyahs: Int,
    val surahNumber: Int,
    val ayahNumber: Int
)

File: iOS/QuranSDK/Models/SectionMetadata.swift
//
//  SectionMetadata.swift
//  QuranSDK
//
//  Created by Adil on 14/06/2019.
//  Copyright © 2019 Tazkiya Tech. All rights reserved.
//

import Foundation

/**
 * Metadata which describes a section of the Quran.
 */
public struct SectionMetadata: Sendable {
    
    public let sectionType: SectionType
    public let sectionNumber: Int
    public let numAyahs: Int
    public let surahNumber: Int
    public let ayahNumber: Int
    
}
